### ISSUE TITLE
UI Internationalization

### ISSUE THREAD

**Issue by turtleizzy** [audreyt/ethercalc#230]
Currently I am working on translating the user interface of ethercalc into Chinese. It will be very convenient for translators if UI strings are saved in a separate file. User can therefore choose their favorite interface language. 
Any gettext-like i18n libraries may be suitable to handle this function, and I personally recommend i18next (i18next.com).

By the way I am more than willing to participate in this translation project.

Update: I didn't notice /l10n/ directory. Sorry for disturbance.

**Comment by audreyt** [comment]
No problem. Please feel free to open a pull request for `zh-CN.json`  — perhaps starting with a sketch in http://opencc.byvoid.com/ — thanks!

**Comment by turtleizzy** [comment]
It seems strings in player-graph.js are not integrated into /l10n/ strings library.

**Comment by audreyt** [comment]
Yes. Help welcome to factor them from line 6-27 out into `l10n/en.json` — we should also have some way of falling back to `en.json` for strings not defined in the local JSON files, please create a ticket for that too if you'd like to work on it; the relevant code is `index.html` line 34.
